packages/sdk-metrics/src/Descriptor.ts

```diff
@@ -32,32 +32,64 @@ export enum InstrumentType {
 /**
  * An interface describing the instrument.
  */
-export interface InstrumentDescriptor {
+export interface Descriptor {
   readonly name: string;
   readonly description: string;
   readonly unit: string;
-  readonly type: InstrumentType;
   readonly valueType: ValueType;
 }
 
-export function createInstrumentDescriptor(
+/**
+ * An interface describing the instrument.
+ * @deprecated Please use {@link Descriptor} instead.
+ */
+export interface InstrumentDescriptor extends Descriptor {
+  /**
+   * The original instrument's type.
+   * @deprecated Any necessary information about a metric's properties is available in the data point.
+   */
+  readonly type: InstrumentType;
+}
+
+/**
+ * An interface describing an actual instrument or virtual instrument (created by views).
+ * Only intended for internal use. For exporting descriptor data, use {@link Descriptor} instead.
+ */
+export interface MetricDescriptor extends Descriptor {
+  /**
+   * The original instrument's type.
+   */
+  readonly type: InstrumentType;
+}
+
+export function toExternal(descriptor: MetricDescriptor): InstrumentDescriptor {
+  return {
+    name: descriptor.name,
+    description: descriptor.description,
+    unit: descriptor.unit,
+    type: descriptor.type,
+    valueType: descriptor.valueType,
+  };
+}
+
+export function createDescriptor(
   name: string,
   type: InstrumentType,
   options?: MetricOptions
-): InstrumentDescriptor {
+): MetricDescriptor {
   return {
     name,
-    type,
+    type: type,
```

Comment:
This change is not necessary.

Reply:
Fixed in [dd7a25f](https://github.com/open-telemetry/opentelemetry-js/pull/3520/commits/dd7a25fa92bd67fe892a6b04d7dbc8609f2972ae) :slightly_smiling_face: